hologram_dashboard/extension/src/app/dashboard.service.ts

```diff
@@ -0,0 +1,16 @@
+export class DashboardService {
+  public static API_ENDPOINT='http://localhost:8000/';
```

Comment:
Just do
export const BACKEND_BASE_URL = 'http://localhost:8080';

No need to wrap it in a class.